test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -44,20 +44,20 @@ namespace iroha {
 
       void SetUp() override {
         AmetsuchiTest::SetUp();
-        sql = std::make_unique<soci::session>(*soci::factory_postgresql(),
-                                              pgopt_);
+        {
+          auto sql = std::make_unique<soci::session>(
+              *soci::factory_postgresql(), pgopt_);
+          prepareTables(*sql);
```

Comment:
Tables are already prepared in `AmetsuchiTest::SetUp()`, in `prepareConnectionPool` call, please remove.